apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -30,14 +32,20 @@ function NotificationList() {
   const { templates, loading: isLoading, totalCount: totalTemplatesCount, pageSize } = useTemplates(page);
   const theme = useMantineTheme();
   const navigate = useNavigate();
-  const {
-    blueprintsGroupedAndPopular: { groupedBlueprints, popularBlueprints } = {},
-    isLoading: areBlueprintsLoading,
-  } = useFetchBlueprints();
-  const hasGroups = groupedBlueprints && groupedBlueprints.length > 0;
+  const { blueprintsGroupedAndPopular: { general, popular } = {}, isLoading: areBlueprintsLoading } =
+    useFetchBlueprints();
+  const { createTemplateFromBlueprint, isLoading: isCreatingTemplateFromBlueprint } = useCreateTemplateFromBlueprint({
+    onSuccess: (template) => {
+      navigate(`${parseUrl(ROUTES.TEMPLATES_EDIT_TEMPLATEID, { templateId: template._id ?? '' })}`);
+    },
+    onError: () => {
+      errorMessage('Something went wrong while creating template from blueprint, please try again later.');
+    },
+  });
+  const hasGroups = general && general.length > 0;
   const hasTemplates = templates && templates.length > 0;
 
-  const { TemplatesStoreModal, openModal } = useTemplatesStoreModal({ groupedBlueprints });
+  const { TemplatesStoreModal, openModal } = useTemplatesStoreModal({ general });
```

Comment:
Never realized that we are not presenting the popular in the Template Store, raised a question [HERE](https://www.figma.com/file/DPyX973ZSrQlszkIQowxXp?node-id=9:6812#451678454)

Reply:
To me, the popular templates are a subset of the general.